connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignorTest.java

```diff
@@ -769,16 +609,16 @@ public void testLostAssignmentHandlingWhenWorkerBounces() {
                 new ArrayList<>(configuredAssignment.values()),
                 memberConfigs);
 
-        assertThat("Wrong set of workers for reassignments",
+        assertEquals("Wrong set of workers for reassignments",
```

Comment:
What about simplifying these assertions into:
```
assertTrue("Wrong set of workers for reassignments", assignor.candidateWorkersForReassignment.isEmpty());
```

Reply:
I usually prefer `assertEquals` when possible since it shows the difference between actual and expected, which in this case would mean the complete set of incorrect candidate workers for reassignment.